src/Auth/Azure.DataApiBuilder.Auth.csproj

```diff
@@ -5,10 +5,9 @@
     <ImplicitUsings>enable</ImplicitUsings>
```

Comment:
Can we use `<AllowedOutputExtensionsInPackageBuildOutputFolder>` to somehow exclude pdbs from the main package?

The example here provides ability to include pdbs in the main package: https://github.com/dotnet/sourcelink#alternative-pdb-distribution, 

similar to that - there should be some way to exclude too as announced here: https://devblogs.microsoft.com/nuget/announcing-nuget-6/#exclude-default-file-extensions

Can you please try searching a bit more?

Reply:
I tried the following approaches for excluding the .pdb files, but unfortunately none of them works 

1. Using output file extensions to include only the types of files we would want in the nuget package

```	
      <PropertyGroup>
         <AllowedOutputExtensionsInPackageBuildOutputFolder>.json;.txt;.png;.md;.dll;.xml;.exe</AllowedOutputExtensionsInPackageBuildOutputFolder>
      </PropertyGroup>
```   

I added this to all the .csproj files that are presnet. But, the `.pdb` files are included in both the primary and symbols Nuget package.  


2. This issue (https://github.com/NuGet/Home/issues/4142) talks about the behavior of `pack` command with regards to creating a separate symbols package. In the disucssion, it was mentioned that there exists a property `IncludeSymbolsInPackage` which when set to `false` would stop the .pdb files from being included in the main Nuget package. I tried setting this property to `false` but that did not stop the `.pdb` files from being included in the primary Nuget package.

3. Using `targetsfortfmspecificbuildoutput` target, I tried deleting the .pdb files as a pre-event for `pack` event, but unfortunately this too does not work. The .pdb files gets included in the primary Nuget package as well.


The size of the Microsoft.DataApiBuilder Nuget package is ~ 16 MB and the size of all the symbols put together is 113 KB. From the perspective of the size of the Nuget package, the presence of the symbol files has very little effect.
 
Considering these, it shouldn't hurt having the symbol files in the primary package even though we publish a separate symbols package

Reply:
Thanks for trying all the other options. Its unfortunate they didn't work. We can include the pdbs in the main package for now since the size concern is not there.